pkg/apis/autoscaling/v1/register.go

```diff
@@ -32,7 +32,6 @@ var SchemeGroupVersion = unversioned.GroupVersion{Group: GroupName, Version: "v1
 func AddToScheme(scheme *runtime.Scheme) {
 	addKnownTypes(scheme)
 	addDefaultingFuncs(scheme)
-	addConversionFuncs(scheme)
```

Comment:
I changed internal types to be identical with v1 so that conversions can be auto-generated and we don't have to have our own functions.